src/Umbraco.Cms.Api.Management/Controllers/Document/References/ReferencesDocumentController.cs

```diff
@@ -28,21 +28,20 @@ public ByIdTrackedReferenceController(ITrackedReferencesService trackedReference
     ///     Used by info tabs on content, media etc. and for the delete and unpublish of single items.
     ///     This is basically finding parents of relations.
     /// </remarks>
-    [HttpGet("{id:guid}")]
+    [HttpGet("{id:guid}/references")]
```

Comment:
`{id}/referenced-by` would be good here - more explicit. I first thought this endpoint returned any items referenced by `id` - but it's the other way around, right?

Reply:
This endpoint should return a list of all items it references, so for a document all the media/content/... that have been picked